setup.py

```diff
@@ -279,17 +279,14 @@ def initialize_options(self):
 
         for root, dirs, files in os.walk('pandas'):
             for f in files:
-                if f in self._clean_exclude:
-                    continue
-
-                # XXX
-                if 'ujson' in f:
```

Comment:
The `ujson` conditional also does nothing because none of the filenames in `src/ujson/lib` have `ujson` in them.